tools/shared.py

```diff
@@ -895,10 +895,10 @@ def get_vanilla_file():
       logging.debug('failed to use vanilla file, will re-check: ' + str(e))
       is_vanilla = check_vanilla()
     temp_cache = None
+    os.environ['EMCC_WASM_BACKEND'] = str(is_vanilla)
```

Comment:
When we spawn child processes in the pool, we want to pass `EMCC_WASM_BACKEND` env var to them (see below), so I wanted to write this out here also in the negative case so that the env var is always present for the child spawns.